test/TestUserAcceptance.hs

```diff
@@ -693,7 +709,10 @@ setUpTempMaildir = do
 -- | run notmuch to create the notmuch database
 -- Note: discard stdout which otherwise clobbers the test output
 setUpNotmuch :: FilePath -> IO ()
-setUpNotmuch notmuchcfg = void $ readProcess "notmuch" ["--config=" <> notmuchcfg, "new"] []
+setUpNotmuch notmuchcfg = let config = setStdin closed
+                                       $ setStdout closed
```

Comment:
Is there a problem with the default process config (inherited stdin/out/err)?
Put another way, what's wrong with:
```haskell
void $ readProcess_ $ proc "notmuch" args
```
(which is the approach taken at most other points in this diff...)

Reply:
I think by default stderr is not closed and it interleaves with the tasty output. I'll check if it was changed by mistake...

Reply:
`readProcess_` creates pipes for the child's stdout and stderr.